labcodes/confetti-storybook
# Migrate narrow sidebar

**Related to:** https://github.com/labcodes/confetti-ds/pull/7

**Link to task:** https://labcodes.atlassian.net/browse/CSYS-200

**Staging app:** https://migrate-narrow-sidebar--confetti-storybook.netlify.app/

**How to test:**
- open the staging app;
- check that the narrow sidebar stories work as described on the task

**Description of your solution:**
Added stories for the Narrow Sidebar and its subcomponents.
